### PR TITLE
Adding Added Mass Condtion and Reservoir Temperature

### PR DESCRIPTION
**Pull request by lorenzogracia**
New Added Mass Condition.
New Reservoir temperature condition, constant temperature (no depth dependency).
Modification of Westergaard process.

### CODE REVIEW

File: applications/DamApplication/custom_conditions/added_mass_condition.cpp [hunk 1]
```diff
@@ -0,0 +1,331 @@
+//   
+//   Project Name:        			KratosDamApplication $
```
Comment (rubenzorrilla) [inline applications/DamApplication/custom_conditions/added_mass_condition.cpp:2]: Use the new header format

**Review by rubenzorrilla — CHANGES_REQUESTED** [review]

File: applications/DamApplication/custom_conditions/added_mass_condition.cpp (hunk 1 above)
Comment (loumalouomega) [reply applications/DamApplication/custom_conditions/added_mass_condition.cpp:2]: @rubenzorrilla I like your spirit

**Review by ipouplana — CHANGES_REQUESTED** [review]
Hi Lorenzo,

I compiled this branch without an error, but when I run the 2d_dam_thermo_mechanical test it fails with the following message:

ModuleNotFoundError: No module named 'impose_bofang_condition_process'.

Probably you forgot to update the test examples...

By the way, @rubenzorrilla could you write here the new header or a file where Lorenzo can find an example of it. I am not sure whether Lorenzo knows how the current header is...

**Comment by rubenzorrilla** [comment]
Hi @ipouplana,

For instance, you can take it from the FluidDynamicsApp/custom_utilities/estimate_dt_utilities.h.

**Comment by ipouplana** [comment]
Thanks @rubenzorrilla

**Review by rubenzorrilla — APPROVED** [review]

**Comment by lorenzogracia** [comment]
Hi @rubenzorrilla, thanks for commenting about the new headers. I have updated the headers in all processes of the dam application.

**Review by ipouplana — APPROVED** [review]

**Comment by lorenzogracia** [comment]
Hi @ipouplana, you were right, I did not update the tests. Now, everything is updated.  I have also changed the reference temperature in the solvers.

**Comment by loumalouomega** [comment]
@lorenzogracia You can go even further an use astyle with the command --style=allman to set all the identation in AllMan

**Comment by loumalouomega** [comment]
@lorenzogracia Additionally you can correct your code using clang-tidy https://github.com/KratosMultiphysics/Kratos/wiki/How-to-use-Clang-Tidy-to-automatically-correct-code

**Comment by lorenzogracia** [comment]
Hi @loumalouomega, thanks for the information, I take it into account.

**Pull request merged**